Examples/Tests/PythonWrappers/PICMI_inputs_2d.py

```diff
@@ -0,0 +1,246 @@
+                       warpx_field_gathering_algo = 'energy-conserving',
+                       warpx_use_filter = 1)
+
+# Add diagnostics to simulation
+sim.add_diagnostic(field_diag)
+
+# Write input file to run with compiled version
+sim.write_input_file(file_name = 'inputs_2d')
+
+# Whether to include guard cells in data returned by Python wrappers
+include_ghosts = 1
+
+# Compute min and max of fields data
+def compute_minmax(data):
+    vmin = -max(abs(data.min()), abs(data.max()))
+    vmax =  max(abs(data.min()), abs(data.max()))
+    return (vmin, vmax)
+
+# Plot fields data either in valid domain or in PML
+def plot_data(data, pml, title, name):
+    fig, ax = plt.subplots(nrows = 1, ncols = 1, gridspec_kw = dict(wspace = 0.5), figsize = [6,5])
+    cax = make_axes_locatable(ax).append_axes('right', size='5%', pad='5%')
+    lw  = 0.8
+    ls = '--'
+    if (pml):
+        # Draw PMLs and ghost regions
+        ax.axvline(x = 0       , linewidth = lw, linestyle = ls)
+        ax.axvline(x = 0+nxg   , linewidth = lw, linestyle = ls)
+        ax.axvline(x = -nxpml  , linewidth = lw, linestyle = ls)
+        ax.axvline(x = nx      , linewidth = lw, linestyle = ls)
+        ax.axvline(x = nx-nxg  , linewidth = lw, linestyle = ls)
+        ax.axvline(x = nx+nxpml, linewidth = lw, linestyle = ls)
+        ax.axhline(y = 0       , linewidth = lw, linestyle = ls)
+        ax.axhline(y = 0+nzg   , linewidth = lw, linestyle = ls)
+        ax.axhline(y = -nzpml  , linewidth = lw, linestyle = ls)
+        ax.axhline(y = nz      , linewidth = lw, linestyle = ls)
+        ax.axhline(y = nz-nzg  , linewidth = lw, linestyle = ls)
+        ax.axhline(y = nz+nzpml, linewidth = lw, linestyle = ls)
+        # Annotations
+        ax.annotate('PML', xy = (-5,64), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML', xy = (133,64), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML', xy = (64,-5), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML', xy = (64,133), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (5,64), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (-13,64), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (125,64), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (143,64), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (64,4), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (64,-15), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (64,124), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (64,142), rotation = 'horizontal', ha = 'center', va = 'center')
```

Comment:
Note: will write the locations of these annotations, and the ones below, in terms of `nx`, `nxg`, `nxpml`, etc.